### PR TITLE
`test`: [M3-6616] - Add tests to update/rename linode label

### PR DESCRIPTION
**Pull request by cliu-akamai**
## Description 📝
Add regression tests for updating/renaming the Linode label.

## Major Changes 🔄
- Add regression tests to update/rename a Linode label

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/linodes/update-linode-labels.spec.ts"
```

### CODE REVIEW

**Review by jdamore-linode — APPROVED** [review]
These tests are great, nice work @cliu-akamai!

File: packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+authenticate();
+describe('update linode label', () => {
+  beforeEach(() => {
+    cleanUp(['linodes']);
+  });
+
+  it('updates a linode label from details page', () => {
+    createLinode().then((linode) => {
+      const newLinodeLable = randomLabel();
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts:16]:
```suggestion
      const newLinodeLabel = randomLabel();
```

File: packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      cy.visitWithLogin(`/linodes/${linode.id}`);
+      containsVisible('RUNNING');
+
+      cy.get(`[aria-label="Edit ${linode.label}"]`).click();
+      cy.get(`[id="edit-${linode.label}-label"]`)
+        .click()
+        .clear()
+        .type(`${newLinodeLable}{enter}`);
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts:24]:
```suggestion
        .type(`${newLinodeLabel}{enter}`);
```

File: packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+      cy.visitWithLogin('/linodes');
+      cy.get(`[data-qa-linode="${newLinodeLable}"]`).should('be.visible');
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts:27]:
```suggestion
      cy.get(`[data-qa-linode="${newLinodeLabel}"]`).should('be.visible');
```

File: packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    });
+  });
+
+  it('updates a linode label from the "Settings" tab', () => {
+    createLinode().then((linode) => {
+      const newLinodeLable = randomLabel();
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts:33]:
```suggestion
      const newLinodeLabel = randomLabel();
```

File: packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      cy.visitWithLogin(`/linodes/${linode.id}`);
+      containsVisible('RUNNING');
+
+      cy.visitWithLogin(`/linodes/${linode.id}/settings`);
+      cy.get('[id="label"]').click().clear().type(`${newLinodeLable}{enter}`);
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts:38]:
```suggestion
      cy.get('[id="label"]').click().clear().type(`${newLinodeLabel}{enter}`);
```

File: packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      ui.buttonGroup.findButtonByTitle('Save').should('be.visible').click();
+
+      cy.visitWithLogin('/linodes');
+      cy.get(`[data-qa-linode="${newLinodeLable}"]`).should('be.visible');
```
Comment (mjac0bs) [inline packages/manager/cypress/e2e/core/linodes/update-linode-labels.spec.ts:42]:
```suggestion
      cy.get(`[data-qa-linode="${newLinodeLabel}"]`).should('be.visible');
```

**Review by mjac0bs — APPROVED** [review]
Thanks Cassie, short and sweet tests! 
Screenshot 2024-01-04 at 2 50 43 PM

All my comments are just to correct one variable typo.

**Review by abailly-akamai — APPROVED** [review]
Nice! Approving pending typos pointed out by @mjac0bs

**Review by carrillo-erik — APPROVED** [review]

**Pull request merged**